agent/Makefile

```diff
@@ -41,7 +41,7 @@ compile-crane: ##
 .PHONY: compile-dagent
 compile-dagent: ##
 	cd cmd/dagent && \
-	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build ${LDFLAGS} -o ../../build/out/dagent; \
+	GOOS=windows GOARCH=amd64 CGO_ENABLED=0 go build ${LDFLAGS} -o ../../build/out/dagent.exe; \
```

Comment:
This might have If this is wanted, you have to a create a new make build target with a different name or even better to set it as a variable/parameter as it is done with **image_version**.
`goos ?= linux` on top, `GOOS=${goos}` here.